airbyte-integrations/connectors/build.gradle

```diff
@@ -99,3 +113,75 @@ allprojects {
+subprojects { project ->
+    // Only apply to connector projects
+    if (project.path.startsWith(':airbyte-integrations:connectors:')) {
+        // Determine connector type during configuration phase
+        def isJavaConnector = project.file('build.gradle').exists()
+        def isPythonConnector = project.file('setup.py').exists() || project.file('pyproject.toml').exists()
+        def connectorType = isJavaConnector ? 'java' : (isPythonConnector ? 'python' : null)
+        if (connectorType == null) {
+            logger.warn("Could not determine connector type for ${project.name}. " +
+                    "Expected either a Java or Python connector.")
+            return // Skip this project
+        }
+        // Set project properties for use in tasks
+        project.ext.connectorType = connectorType
+        // Task to build connector Docker image
+        def buildConnectorDockerImage = project.tasks.register('buildConnectorDockerImage', Exec) {
+            description = 'Build connector Docker image using existing Dockerfiles'
+            group = 'build'
+            // For Java connectors, depend on distTar task
+            if (connectorType == 'java') {
+                dependsOn project.tasks.matching { it.name == 'distTar' }
+            }
+            workingDir = rootProject.file("docker-images")
+            doFirst {
+                logger.lifecycle("Building Docker image for ${project.name} (${connectorType} connector)")
+                // Get native architecture
+                def nativeArch = "amd64"
+                try {
+                    def archProcess = "uname -m".execute()
+                    archProcess.waitFor()
+                    def arch = archProcess.text.trim()
+                    if (arch == "arm64" || arch == "aarch64") {
+                        nativeArch = "arm64"
+                    }
+                } catch (Exception e) {
+                    logger.warn("Failed to detect native architecture: ${e.message}. Defaulting to amd64.")
+                }
+                logger.lifecycle("Detected native architecture: ${nativeArch}")
+                // Determine base image based on connector type
+                def baseImage = connectorType == 'java' ? 'airbyte/java-connector-base:2.0.1' : 'airbyte/python-connector-base:1.1.0'
+                // Build Docker image using the existing Dockerfile
+                commandLine 'docker', 'build',
+                    '--platform', "linux/${nativeArch}",
+                    '--label', 'io.airbyte.version=3.11.15',
+                    '--label', "io.airbyte.name=airbyte/${project.name}",
+                    '--file', "Dockerfile.${connectorType}-connector",
+                    '--build-arg', "BASE_IMAGE=${baseImage}",
+                    '--build-arg', "CONNECTOR_NAME=${project.name}",
+                    '-t', "airbyte/${project.name}:dev",
+                    project.projectDir
```

Comment:
I thought you were going to use the plugin to build this. It still looks like a cli exectution.